src/test/regress/expected/with_nested.out

```diff
@@ -0,0 +1,381 @@
+CREATE TABLE local_users (user_id int, event_type int);
```

Comment:
Random note: Shall we add a test such that different CTEs uses tables from different schemas? Public schema + one of the existing should suffice to test that?